libs/checkpoint-postgres/langgraph/checkpoint/postgres/aio.py

```diff
@@ -288,27 +291,38 @@ async def aput_writes(
                 ),
             )
 
+    @asynccontextmanager
+    async def _connection(self) -> AsyncIterator[AsyncConnection]:
```

Comment:
Why do we want one more private method that's used only in one place

Reply:
moved outside the class but lmk if that's not what you wanted!